README.md

```diff
@@ -1,30 +1,32 @@
-# Project Migrator 
+# Project Migrator
 
 ### Requirements
 
-- You must have Deno installeds
+- You must have [Deno](https://deno.land/) installed. If you use Homebrew, run `brew install deno`.
 
 
 ### Considerations
 
-- These scripts (migrate and source) are supported strictly as is and LD support cannot be expected for help running this
+- These scripts, `migrate.ts` and `source.ts`, are provided strictly as-is. LaunchDarkly Support cannot be expected for help running this.
 
-### Known Issues
+### Known issues
 
 - Bug with 429 rate limiting. The request that is rate limited will still fail
```

Comment:
```suggestion
- Bug with 429 rate limiting. The rate-limited request will still fail,
```
I don't have much context for this section, so feel free to ignore any suggestions that don't make sense.